requirements.txt

```diff
@@ -58,6 +59,7 @@ MarkupSafe==1.1.1
 mccabe==0.6.1
 mypy==0.790
 mypy-extensions==0.4.3
+nest-asyncio==1.5.1
```

Comment:
You sure you need this?

Reply:
I think so. My tests are problematic if I don't use it.